### ISSUE TITLE
How should the quantum circuit be integrated with the classical style encoder?

### ISSUE THREAD

**Issue by CYeryeong** [AIMLAB-QuantumAI/Quantum_Style_Transfer#5]
I’m trying to **connect the quantum layer to the classical-style encoder network**, but I’m not sure about the correct interface. Any design suggestions or code references from the original implementation would be appreciated.

**Comment by CYeryeong** [comment]
I looked into this further and managed to get the quantum layer connected properly. The key was that the quantum module should process a compact latent vector, not the full feature map from the encoder. I added a linear projection layer after the classical-style encoder to reduce the latent dimension to match the number of qubits. Also, normalizing the latent values to the range [-1, 1] helped stabilize gate parameter updates during training.


latent = style_encoder(input_image)
reduced = nn.Linear(latent_dim, num_qubits)(latent)
quantum_out = quantum_layer(reduced)
output = decoder(quantum_out)
